addOns/sstiscanner/src/main/resources/org/zaproxy/zap/extension/sstiscanner/resources/Messages.properties

```diff
@@ -0,0 +1,17 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+sstiscanner.name = Zap-ESUP scanner for SSTI
+sstiscanner.desc = This is a scanner for SSTI  in the form of reflected, stored and blind.
```

Comment:
There's a double space after SSTI. The list of forms should have a comma (oxford/serial comma) before the and. (Stored and blind aren't a single entity they're separate).